src/types.ts

```diff
@@ -41,6 +41,7 @@ export interface QueriedSong {
     tags?: string;
     language?: string;
     vtype?: string;
+    selectionWeight?: number;
```

Comment:
Can this be made required?

Reply:
I don't know why any of these are nullable tbh